x-pack/packetbeat/scripts/mage/pcap.go

```diff
@@ -84,7 +84,7 @@ func GetNpcapInstallerFn(dir string) func() error {
 		ciBucketName := getBucketName()
 
 		fmt.Printf("getting %s from private cache to %q\n", installer, dstPath) //nolint:forbidigo // fmt.Println is ok here
-		return sh.RunV("gsutil", "cp", "gs://"+ciBucketName+"/private/"+installer, dstPath)
+		return sh.RunV("gcloud", "storage", "cp", "gs://"+ciBucketName+"/private/"+installer, dstPath)
```

Comment:
For the record `gsutil` does not honour the Env variables for the GCP created on the fly, but `gcloud` does.

Reply:
> For the record `gsutil` does not honour the Env variables for the GCP created on the fly, but `gcloud` does.

Good to know; my understanding is that `gcloud storage cp` is the more modern and scalable way to do things, so this is better as well.